lib/classes/episode_link.rb

```diff
@@ -4,15 +4,15 @@
 require_relative '../pages/episode_page'
 
 class EpisodeLink < Struct.new(:href)
-  def call
+  def call(browser)
```

Comment:
For this I would rather keep arguments in the struct so that we keep a consistent api across classes (sorry for being nitpicky on such a small project).  If you want to pair up to knock this out let me know, i can also do a branch off yours to suggest differences.